app/generate/page.tsx

```
@@ -0,0 +1,118 @@
+            required
+          >
+            <option value="">Select Experience Level</option>
+            <option value="Entry Level">Entry Level (0-2 years)</option>
+            <option value="Mid Level">Mid Level (3-5 years)</option>
+            <option value="Senior Level">Senior Level (5+ years)</option>
+            <option value="Lead">Lead (7+ years)</option>
+          </select>
+        </div>
+
+        <div>
+          <label className="block font-medium mb-1">Job Details</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)